### PR TITLE
Release notes for 128T 4.2.8

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

File: docs/release_notes_128t_4.2.md [hunk 1]
```diff
@@ -3,6 +3,52 @@ title: 128T 4.2 Release Notes
...
+- **I95-33842** Race condition on 128T startup, causing DHCP server to fail to start
+  _**Conditions:**_ DHCP server is not running. The following log message can be seen:
+  ```
+init[5720]: [dh00000001 | dhcp-server-ns-1:1073742075] Running command ['/usr/sbin/ip', 'netns', 'set', 'dhcp-server-ns-1', '1073742075']
+init[5720]: [dh00000001 | dhcp-server-ns-1:1073742075] Command "/usr/sbin/ip netns set dhcp-server-ns-1 1073742075" failed: RTNETLINK answers: No space left on device
+  ```
+  Until the system is upgraded to 4.2.8, this issue can be mitigated restarting the 128T process.
+------
+- **I95-34053** When configured to use LDAP, locally created user credentials and access are not honored for those users that already exist in LDAP.
+  Until the system is upgraded to 4.2.8, this issue can be mitigated by rebooting the 128T.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.2.md:19]: rebooting or restarting?

File: docs/release_notes_128t_4.2.md [hunk 2]
```diff
@@ -3,6 +3,52 @@ title: 128T 4.2 Release Notes
...
+- **I95-35093** `show asset <asset-id>` incorrectly continues to show `Currently Upgrading` version after completion of an upgrade.
+------
+- **I95-35099** Removing a 128T user does not remove its Linux credentials, allowing the user to still login to Linux.
+  Until the system is upgraded to 4.2.8, this issue can be mitigated by disabling rather than deleting the user.
+------
+- **I95-35115** Aggregate bandwidth charts may not display data accurately
+------
+- **I95-35155** `show device-interface` output did not include duplex mode
+------
+- **I95-35188** Adding a tenant or changing the order of tenant in the configuration can lead to traffic being dropped upon session recovery
```
Comment (ptimmons) [inline docs/release_notes_128t_4.2.md:47]: changing the order of tenants

File: docs/release_notes_128t_4.2.md (hunk 1 above)
Comment (MichaelBaj) [reply docs/release_notes_128t_4.2.md:19]: restart

File: docs/release_notes_128t_4.2.md [hunk 3]
```diff
@@ -3,6 +3,52 @@ title: 128T 4.2 Release Notes
...
+
+### Issues Fixed
+
+- **I95-33842** Race condition on 128T startup, causing DHCP server to fail to start
+  _**Conditions:**_ DHCP server is not running. The following log message can be seen:
+  ```
+init[5720]: [dh00000001 | dhcp-server-ns-1:1073742075] Running command ['/usr/sbin/ip', 'netns', 'set', 'dhcp-server-ns-1', '1073742075']
+init[5720]: [dh00000001 | dhcp-server-ns-1:1073742075] Command "/usr/sbin/ip netns set dhcp-server-ns-1 1073742075" failed: RTNETLINK answers: No space left on device
+  ```
+  Until the system is upgraded to 4.2.8, this issue can be mitigated restarting the 128T process.
```
Comment (jeffmart) [inline docs/release_notes_128t_4.2.md:16]: nit: change "mitigated restarting" to "mitigated by restarting"

**Pull request merged**